packages/core/src/animation/Animator.ts

```diff
@@ -482,7 +482,9 @@ export class Animator extends Component {
     const { srcPlayData, destPlayData, crossFadeTransition: crossFadeTransitionInfo } = animLayerData;
     const layerAdditive = blendingMode === AnimatorLayerBlendingMode.Additive;
     const layerWeight = firstLayer ? 1.0 : weight;
-    this._checkTransition(srcPlayData, crossFadeTransitionInfo, layerIndex);
+    animLayerData.layerState !== LayerState.FixedCrossFading &&
+      this._checkTransition(srcPlayData, crossFadeTransitionInfo, layerIndex);
+
```

Comment:
- First of all, I want to confirm the environmental conditions that need to call `_checkTransition`, if `LayerState.CrossFading` also needs.
- Maybe we should fire the script events within range( lastFrameTime - transitionTime), and then do transition.